cve_bin_tool/checkers/sqlite.py

```diff
@@ -151,8 +181,8 @@ def get_version_map():
         print("Could not fetch " + changeurl + ", " + err)
     lines = response.readlines()
 
-    version_pattern = re.compile(r'<h3>\d{4}-\d{2}-\d{2} \((\d+\.\d+\.?\d?)\)</h3>')
-    id_pattern = re.compile(r'SQLITE_SOURCE_ID: "([^"]+)"')
+    version_pattern = re.compile(r'<h3>\d{4}-\d{2}-\d{2} \((\d+\.\d+\.\d+[.\d]*)\)</h3>')
+    id_pattern = re.compile(r'"*(\d{4}-\d{2}-\d{2} \d+:\d+:\d+ [\w]+)"*')
```

Comment:
Please keep the SQLITE_SOURCE_ID (it's what the project has been using since 2009, and we may want to get it out of source for the ones before 2011 so there's no benefit to generalizing the pattern when we need a new data source).   Also, something here is broken because we're losing version 3.14 (which is a real version).  But thanks for picking up the missing x.x.x.x stuff; I hadn't even filed a ticket for it yet.